src/ParseServer.js

```diff
@@ -56,6 +56,7 @@ import DatabaseController       from './Controllers/DatabaseController';
 const SchemaController = require('./Controllers/SchemaController');
 import ParsePushAdapter         from 'parse-server-push-adapter';
 import MongoStorageAdapter      from './Adapters/Storage/Mongo/MongoStorageAdapter';
+import { DirectRESTController } from './DirectRESTController';
```

Comment:
When not using {} this is to import the `default`, with {} it's the spread operator to get the properties of the exported object.